sdk/file_sdk/UnitTests/zip_ut/zip_tests.cpp

```diff
@@ -0,0 +1,72 @@
+#include <QString>
+#include <QtTest>
+#include <Qt>
+#undef signals
+#include "../../OPCFileFormat/zip_file.h"
+#include <iostream>
+#include <QTStreamImpl.h>
+#include <memory>
+#include <CryptoAPI.h>
+#include <sstream>
+#include <fstream>
+#include <stdio.h>
+#include <Common/std_stream_adapter.h>
+
+class ZipTests : public QObject
+{
+    Q_OBJECT
+
+public:
+    ZipTests();
+
+private Q_SLOTS:
+    void testCaseGetEntryReturnsXml();
+    void testCaseMissingEntryThrowZipException();
+    void testCaseNotAZipThrowZipException();
+};
+
+using mip::file::IStream;
+using std::static_pointer_cast;
+using std::make_shared;
+using std::shared_ptr;
+using std::string;
+using mip::file::ZipFile;
+using mip::file::StdStreamAdapter;
+
+const string notLabeledInputFilePath = "\\\\sislands\\Public\\RnD\\rms-sdk\\Tests\\NotLabeled.docx";
+const string labeledInputFilePath = "\\\\sislands\\Public\\RnD\\rms-sdk\\Tests\\LabeledGeneral.docx";
+const string outputFilePath = "\\\\sislands\\Public\\RnD\\rms-sdk\\Tests\\Output.docx";
+
```

Comment:
why global? define in class or in namespace

Reply:
and on one machine in other company which use the sdk...

---
In reply to: [139360341](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139360341) [](ancestors = 139360341)